examples/example-app-v12/src/app/demo/async-helper.spec.ts

```diff
@@ -0,0 +1,204 @@
+import { fakeAsync, tick, waitForAsync } from '@angular/core/testing';
+import { jest } from '@jest/globals';
+import { DoneFn } from '@jest/types/build/Circus';
```

Comment:
we should not import from `build` which is not recommended by Jest team. Would you please also correct other places?